src/AllBookings/BookingCard/BookingCard.js

```diff
@@ -78,6 +78,7 @@ const BookingCard = (props: Props) => {
   const databaseId = idx(props.booking, _ => _.databaseId);
   const passengerCount = idx(props.booking, _ => _.passengerCount);
   const status = props.booking.status && bookingStatus[props.booking.status];
+  const bookingDate = idx(props.booking, _ => _.bookingDate);
```

Comment:
Shouldn't this field `bookingDate`, together with other (unrelated to this PR) fields such as `passengerCount` and `status` be retrieved from `DateAndPassenger`'s fragment? I mean `BookingCard` component doesn't seem to use them.

Reply:
yep, that's true, my bad, it can be an issue a part if you want @edgarordonez ?

Reply:
Okay, Do I create a new issue, right? @git-toni

Reply:
That would be ideal I guess :slightly_smiling_face:.

I'm also thinking we should evaluate the `null`ability of those fields, and maybe provide `idx(...) | something|`? Except `bookingDate`, which I believe cannot be `null` nor empty being of `GraphQLDate` type?

Reply:
Okay, so I'm gonna to create a new issue to solve it. 😃

Reply:
>  Except bookingDate, which I believe cannot be null

can be null as well, nearly everything can be null in GraphQL 💥